src/pages/nonprofit/issues/[id].tsx

```diff
@@ -1,9 +1,218 @@
+              </HStack>
+            </HStack>
+            {issue.status == IssueStatus.PENDING && (
+              <Link href={urls.pages.nonprofit.issues.index} passHref={true}>
+                <Button variant="secondary" fontSize="sm" onClick={closeIssue}>
+                  Cancel Request
+                </Button>
+              </Link>
+            )}
+          </HStack>
+
+          <VStack
+            maxW="1000px"
+            border="1px solid"
+            borderColor="border"
+            borderRadius="lg"
+            backgroundColor="surface"
+            align="stretch"
+          >
+            <VStack align="stretch" p="20px" spacing={5}>
+              <VStack align="stretch">
+                <Text fontSize="sm" fontWeight="bold">
+                  Issue #
+                </Text>
+                <Text fontSize="sm">{issue._id}</Text>
+              </VStack>
+              <VStack align="stretch">
+                <Text fontSize="sm" fontWeight="bold">
+                  Issue Type
+                </Text>
+                <Text fontSize="sm">{issue.type}</Text>
+              </VStack>
+              <VStack align="stretch">
+                <Text fontSize="sm" fontWeight="bold">
+                  Description
+                </Text>
+                <Text fontSize="sm">{issue.description}</Text>
+              </VStack>
+              <VStack align="stretch">
+                <Text fontSize="sm" fontWeight="bold">
+                  Screenshots
+                </Text>
+                <HStack>
+                  {issue.images.map((image, index) => (
+                    <VStack key={index} align="start">
+                      {/* <Image boxSize="100px" src={image.blobPath} /> */}
+
+                      <Image
+                        boxSize="100px"
+                        src={image}
+                        //src={linkToUploadedFile(image.blobPath)}
+                        onClick={onOpen}
+                        cursor="pointer"
+                      />
+
+                      <Modal isOpen={isOpen} onClose={onClose}>
+                        <ModalOverlay />
+                        <ModalContent margin="auto">
+                          <ModalBody>
+                            <Image boxSize="100%" src={image} />
+                            {/* <Image boxSize="100%" src={linkToUploadedFile(image.blobPath)}/> */}
+                          </ModalBody>
+                        </ModalContent>
+                      </Modal>
+                      <Text fontSize="xx-small">{index}</Text>
+                    </VStack>
+                  ))}
+                </HStack>
+
+                <HStack />
+              </VStack>
+              {hasReviewer && (
+                <VStack align="stretch">
+                  <Text fontSize="sm" fontWeight="bold">
+                    Who To Contact
+                  </Text>
+                  <Text fontSize="sm">{(issue.reviewer as User).name}</Text>
```

Comment:
Minor thing, but you could define a variable outside of the JSX and set it equal to "issue.reviewer as User".  That way you can access the properties here while casting once instead of three times.